src/components/Flex/Flex.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { css } from "emotion";
+
+type Props = {
+  children: React.ReactNode;
+  direction?: "row" | "column";
+  justify?: "flex-start" | "space-between"; // Extend as required
+  fullWidth?: boolean;
+};
+
+export const Flex = ({
+  children,
+  direction = "row",
+  justify = "flex-start",
+  fullWidth = false
```

Comment:
My preference here would be to not try to contain all flex styling into one abstraction. If you're going to build a library of abstractions it's better to keep them declarative and readable by using smaller components like `Row` and `Column`.

This creates better boundaries and also guards against `Flex` becomming a bucket for all styles that involve `display: flex` which is a lot

Reply:
Also, there are five instances where we already have `<Row>` declared in the code and four with `<Column.`. This strongly suggests there's a use case for abstracting that pattern

Reply:
The logic of both components are near identical, having them as one component would avoid us from having to update 2 components when we add a new prop (but granted it is likely we wont be updating this regularly).

Also this is already implemented in DCR.

I feel this is more just a question of taste. So @gtrufitt would you have a preference?

Reply:
My preference would be maybe to publish the component from DCR and import it into discussion rendering but that's probably not what you wanna here :D. Any case, consistency between the two is probably the most useful.

Reply:
Which is more readable 🤷‍♂ 

```
            <Flex direction="column">
              <Flex direction="row">
                <div>Thing</div>
              </Flex>
              <Flex direction="row">
                <Flex direction="column">
                  <div>Thing</div>
                  <div>Thing</div>
                  <div>Thing</div>
                </Flex>
              </Flex>
            </Flex>
```

```

            <Column>
              <Row>
                <div>Thing</div>
              </Row>
              <Row>
                <Column>
                  <div>Thing</div>
                  <div>Thing</div>
                  <div>Thing</div>
                </Column>
              </Row>
            </Column>
```

Reply:
you have convinced me, `<Row>` / `<Column>` it is

Reply:
If we do implement this new Flex to shorthandfor  rows and columns then we should definately not do it and then leave the other pattern in the codebase as well. If we do  replace 'Row' and 'Column' then I'd much rather have it done properly rather than some places doing it one way and others doing it another

Reply:
we use Flex & Row/Column are in DCR . I feel like this was done locally just to help readability, but as a result it wasn't done with a lot of for-sight, we should just agree on a common abstraction, regardless of it is one or the other